packages/zoe/src/contractSupport/safeMath.js

```diff
@@ -19,5 +19,26 @@ export const natSafeMath = harden({
     y = Nat(y);
     return Nat(Nat(x) + y - 1n) / y;
   },
-  isGTE: (x, y) => x >= y,
+  /**
+   * Divide using half-to-even (aka Banker's Rounding) as in IEEE 774 default rounding
+   *
+   * @param {NatValue} a
+   * @param {NatValue} b
+   */
+  bankersDivide: (a, b) => {
+    a = Nat(a);
+    b = Nat(b);
+
+    const div = a / b;
+    const rem = a % b;
+    // if remainder > half divisor, should have rounded up instead of down, so add 1
+    if (rem * 2n > b) {
+      return div + 1n;
+    } else if (rem * 2n === b) {
+      // Add 1 if result is odd to get an even return value
+      if (div % 2n === 1n) return div + 1n;
+    }
+    return div;
+  },
+  isGTE: (x, y) => Nat(x) >= Nat(y),
```

Comment:
Care to comment on the drive-by `isGTE` change?  It seems obvious, but I'd like to know the rationale.

Reply:
To fulfill the description of `natSafeMath` that it validates inputs.
https://github.com/Agoric/agoric-sdk/pull/5233/files/ac455010fe9dabd43442e37b428d9610b63b2ce5#diff-20e0c0ea726a6a4fa25ba5d4940669a675363702b47979f1b8dadeb8c731dfcaR9-R10